src/zope/sqlalchemy/README.rst

```diff
@@ -57,6 +57,31 @@ setting the TEST_TWOPHASE variable to a non empty string. e.g:
 
 $ TEST_DSN=postgres://test:test@localhost/test TEST_TWOPHASE=True bin/test
 
+Wrap SQLAlchemy sessions
+========================
+
+.. code-block:: python
```

Comment:
This could use some text explaining what the code block is doing and why you would want to look at it.

(And also what "wrap SQLAlchemy session" means and why you would need to do that.)